.circleci/config.yml

```diff
@@ -1127,22 +1127,30 @@ jobs:
     environment:
```

Comment:
I think you need to add these to `split-e2e-tests.ts` that will generate this file based on `config-base.yml` otherwise these will be lost.

Reply:
this is generated by the split script. The script will automatically add any files in the __tests__ dir